conf.py

```diff
@@ -58,6 +58,8 @@
     'sphinx_copybutton',
     "sphinx.ext.intersphinx",
     "sphinx.ext.autodoc",
+    'sphinx.ext.autosectionlabel',  # automatically generate targets for
+                                    # each section header
```

Comment:
I think these inline comments can be removed. A quick Internet search reveals its purpose and usage.